### PR TITLE
Feat : Added filters to product list page.

### PR DESCRIPTION
**Pull request by zoef-yt**
Added filters to the product list page.

### CODE REVIEW

File: screens/product-listing-page/product-listing-page.css [hunk 1]
```diff
@@ -12,30 +13,78 @@
...
+}
+
+.dropdown-content {
+	display: none;
+	background-color: var(--colour-accent-second);
+	border: 1px solid var(--colour-accent-one);
+	color: var(--colour-text-white-two);
+	border-radius: 0 0 1rem 1rem;
+	position: absolute;
+	min-width: 160px;
```
Comment (itsmebipul00) [inline screens/product-listing-page/product-listing-page.css]: REMS??

Comment (bhushannaikaroskar) [reply screens/product-listing-page/product-listing-page.css]: +1 rems?

**Review by itsmebipul00 — APPROVED** [review]

**Comment by itsmebipul00** [comment]
Nice work man!

**Review by bhushannaikaroskar — APPROVED** [review]
Apart from that rems looks good to me. The hover part is really good I liked it. Good to merge

**Review by j24m — APPROVED** [review]
Code seems fine and readable to me, UI looks amazing as usual, great work:)
GTM!!!

**Review by zoyatanweer — APPROVED** [review]
This looks superb! UI is just fab. Good to merge.

**Review by varun13169 — APPROVED** [review]
Looks good to me.

File: screens/product-listing-page/product-listing-page.html [hunk 2]
```diff
@@ -252,5 +315,6 @@ <h5>PS4</h5>
 		<footer class="homepage-footer">
 			<p>Made with &hearts; by zoef shaikh</p>
 		</footer>
+		<script src="/app.js"></script>
```
Comment (sanjitsarkar) [inline screens/product-listing-page/product-listing-page.html:318]: You can use script tag inside head tag by using attribute defer
<script src="/app.js" defer></script>

**Pull request merged**